src/CSnakes.Runtime/CPython/Import.cs

```diff
@@ -18,6 +18,27 @@ internal static PyObject Import(string name)
         return PyObject.Create(module);
     }
 
+    internal static PyObject Import(string name, string code, string path)
+    {
+        
+        nint codeObject = Py_CompileString(code, path, InputType.Py_file_input);
+        if (codeObject == IntPtr.Zero)
+        {
+            throw PyObject.ThrowPythonExceptionAsClrException();
+        }
```

Comment:
This error case logic is already handled by [`PyObject`'s constructor and `Create`](https://github.com/tonybaloney/CSnakes/blob/35db15a98c5a4b604b900fabb9325909277b103e/src/CSnakes.Runtime/Python/PyObject.cs#L16-L28), so why not just do `PyObject.Create` on the returned value?